flight_phase_files/HALO_RF04_20200126_info.yaml

```
@@ -5,12 +5,17 @@ flight_id: HALO-0126
 contacts:
 - name: Marc Prange
   email: marc.prange@uni-hamburg.de
-  tags:
-  - dataset creator
+  tags:	  
```

There were some characters behind "tags:" that caused the reports scrip to provide an error message
```suggestion
  tags:
```